modules/programs/direnv.nix

```diff
@@ -121,8 +121,12 @@ in
   config = mkIf cfg.enable {
     home.packages = [ cfg.package ];
 
-    xdg.configFile."direnv/direnv.toml" = mkIf (cfg.config != { }) {
-      source = tomlFormat.generate "direnv-config" cfg.config;
+    xdg.configFile."direnv/direnv.toml" = mkIf (cfg.config != {} || cfg.silent) {
+      source = tomlFormat.generate "direnv-config" cfg.config
+        // (lib.mkIf cfg.silent {
+          log_format = "-";
+          log_filter = "^$";
```

Comment:
These are supposed to be under the global attribute.

Reply:
oh your're right, event though it works, I'll change it accordingly..